recipe/meta.yaml

```
@@ -32,32 +34,37 @@ requirements:
     - cmor
     - json-c
     - libuuid
+    - magics
   run:
+    - llvm-openmp  # [osx]
     - jasper
     - libnetcdf
+    - hdf5
     - proj
     - eccodes
     - udunits2
     - libxml2
     - fftw
     - cmor
     - libuuid
+    - magics
 
 test:
   files:
     - dummy.nc
   commands:
     - cdo -h
     - cdo sinfo dummy.nc
+    - cdo -graph,device=png,linewidth=1 -settaxis,2000-01-01,12:00:00,1days -expr,'t=sin(seq*3.141529/180.0)' -for,1,6000 t
 
 about:
   home: https://code.zmaw.de/projects/cdo
   license: GPL-2.0
```

I think the linter complains because it is now necessary to be explicit about whether later versions of the GPL are allowable or not by using the more concrete 'GPL-2.0-only' or 'GPL-2.0-or-later' SPDX identifiers.

**More importantly**, the license is straight-up wrong: Since 2.0 CDO is under 3-clause BSD license, from 'NEWS':
```
Version 2.0.0 (29 October 2021):

   New features:
     * Changed to C++14
     * Changed to 3-clause BSD license
```

hence:
```suggestion
  license: BSD-3-Clause
```

---

@zklaus Thank you for the review! I changed the specified license.